src/server/server.ts

```diff
@@ -52,13 +49,17 @@ function checkSupport(content: Content): Supported {
 
 }
 
+function getArticleSubtype(capi: Content) {
+  if (pillarFromString(capi.pillarId) === Pillar.opinion) return OpinionArticle;
+  if (capi.fields.displayHint === 'immersive') return ImmersiveArticle;
```

Comment:
Can we prefer explicit block statements for `if`? Leaving them out can lead to bugs, as described [here](https://eslint.org/docs/rules/curly).